src/software/ai/hl/stp/tactic/dribble/dribble_fsm.h

```diff
@@ -214,4 +214,5 @@ struct DribbleFSM
     // the dribble tactic config
     TbotsProto::DribbleTacticConfig dribble_tactic_config;
     Point continuous_dribbling_start_point;
+    static constexpr double INTERCEPTION_POINT_ADJUSTMENT_M = 0.05015;
```

Comment:
The 0.00015 meter increase seems like a hack. Curious why we needed it? If it's actually needed, could you add a comment explaining it as well?